.eslintrc.json

```diff
@@ -9,7 +9,8 @@
   },
   "extends": [
     "eslint:recommended",
-    "prettier" // ALWAYS LAST: disable style rules that conflict with prettier
+    "prettier", // ALWAYS LAST: disable style rules that conflict with prettier
+    "plugin:cypress/recommended"
```

Comment:
Sounds like a good addition! But now it's _last_ in the last, after "prettier" where we have a comment stating it should always be last... 😉  I wrote that comment, but in retrospect, it's probably always supposed to be _after_ "eslint:recommended" since I think Prettier's rules probably only know about basic/built-in ESLint rules. They can't be monitoring every ESLint plugin in existence...

So would you mind updating the "ALWAYS LAST" statement to "AFTER ESLINT" as part of this change?

Reply:
That's a fair point, though I think **AFTER ESLINT** feels somehow unclear since every plugin is for eslint.

I am just wondering if it's still better to keep `prettier` at last, so it is always the bottom line of coding style regardless whatever plugin inserted before `prettier`. In this case I can move `prettier` to last, how do you think?

Reply:
> That's a fair point, though I think **AFTER ESLINT** feels somehow unclear since every plugin is for eslint.
> 
> I am just wondering if it's still better to keep `prettier` at last, so it is always the bottom line of coding style regardless whatever plugin inserted before `prettier`. In this case I can move `prettier` to last, how do you think?

I just changed `prettier` to last, see if you think the other way around. I am okay to change it back. 🥂

Also I added the only missing `allowClickOutside` test now with all tests in place I should feel safer to help review and enhance the library 🎉

Reply:
Sounds good on both points, thanks!